generators/app/templates/gulpfile.js

```diff
@@ -3,20 +3,21 @@ var del = require('del');
 var babel = require('gulp-babel');
 
 gulp.task('default', function() {
-    // babel src js files
-    gulp.src('./src/*.js')
-       .pipe(babel())
-       .pipe(gulp.dest('./lib'));
+  // babel src js files
+  gulp
+    .src('./src/*.js')
+    .pipe(babel())
+    .pipe(gulp.dest('./lib'));
 
-    // babel src component js files
-    gulp.src('./src/components/*.js')
-       .pipe(babel())
-       .pipe(gulp.dest('./lib/components'));
+  // babel src component js files
+  gulp
+    .src('./src/components/*.js')
+    .pipe(babel())
+    .pipe(gulp.dest('./lib/components'));
 
-    // copy sass files to lib
-    gulp.src('./src/styles/*.scss')
-       .pipe(gulp.dest('./lib/styles'));
+  // copy sass files to lib
+  gulp.src('./src/styles/*.scss').pipe(gulp.dest('./lib/styles'));
```

Comment:
Nit: Why is this set of commands formatted differently than the ones above it?

Reply:
hmm that must have been a prettier/linting thing. Since the line length wasn't too long it likes to fit it on one line i think.